packages/lib/server/queries/booking/index.ts

```diff
@@ -0,0 +1,23 @@
+import prisma from "@calcom/prisma";
+
+export const getTotalBookingDuration = async ({
+  eventId,
+  startDate,
+  endDate,
+}: {
+  eventId: number;
+  startDate: Date;
+  endDate: Date;
+}) => {
+  // Aggregates the total booking time for a given event in a given time period
+  const [totalBookingTime] = (await prisma.$queryRaw`
```

Comment:
totalMinutes in this case should be the same as booking.eventType.length so this raw query it's not required I think.